third-party/components/ApprovalCommentForm/ApprovalCommentForm.scss

```diff
@@ -0,0 +1,137 @@
+@import '../../../src/variables';
+
+.bcs-comment-input-container {
```

Comment:
Are all of these styles still necessary if we're using BRUI? If so, can they be refactored as part of this process?

Reply:
if we're using BRUI, we can generalize these styles but right now they're prefixed in the Elements codebase and have very different classnames in the old version that's still in BRUI